specs/core/0_beacon-chain.md

```diff
@@ -1682,7 +1661,7 @@ def process_deposit(state: BeaconState, deposit: Deposit) -> None:
     assert verify_merkle_branch(
         leaf=hash_tree_root(deposit.data),
         proof=deposit.proof,
-        depth=DEPOSIT_CONTRACT_TREE_DEPTH,
+        depth=DEPOSIT_CONTRACT_TREE_DEPTH + 1,
```

Comment:
`+1` due to the length mixin? Does that deserve a comment?

Reply:
Yes, length mixin

Reply:
Added a small comment for it in 8d2cbc9